tools/tplgfw-sanity-check.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+import argparse
+import construct
+import enum
+import os
+import re
+import sys
+import typing
+import uuid
+from construct import this, Container, ListContainer, Struct, Switch, Array, Bytes, PaddedString
+from construct import Byte, Int16ul, Int32ul, Int64ul
+import tplgtool2
+
```

Comment:
docstring or readme to explain usage and functionality

Reply:
add help information to explain usage